### ISSUE TITLE
yflip=true is not working in heatmap plot in GR

### ISSUE THREAD

**Issue by juliohm** [JuliaPlots/Plots.jl#1491]
The option stopped working, at least with the GR backend:

```julia
heatmap([i for i=1:10, j=1:10], yflip=true)
```

**Comment by pkofod** [comment]
Also spy I believe (tried yesterday)

**Comment by daschw** [comment]
```julia
heatmap([i for i=1:10, j=1:10], yflip=false)
```

**Comment by juliohm** [comment]
@daschw `yflip=false` produces the same output here.

**Comment by daschw** [comment]
Oh, my bad, for some reason I thought it would default to `yflip = true`.
However, `yflip` seems to work for me on Plots master with GR 0.31.0:
```julia
heatmap([i for i=1:10, j=1:10])
```
![heatmap_noflip](https://user-images.githubusercontent.com/16589944/39009740-ccbd76f6-440c-11e8-940e-f4a62a9e15cd.png)

```julia
heatmap([i for i=1:10, j=1:10], yflip=true)
```
![heatmap_flip](https://user-images.githubusercontent.com/16589944/39009764-d677921c-440c-11e8-9fab-3fd9f7bb4a1d.png)

Is this not the expected output?

**Comment by juliohm** [comment]
Yes, the output looks correct. The behavior is inverted from previous Plots.jl versions. Maybe a new tag would solve the issue. :+1:

**Comment by pkofod** [comment]
I have Plots v0.17 and GR v0.31 and I don't get the inverted order of the yaxis in the second panel of your example. I get two identical figures looking like your first plot.

**Comment by mkborregaard** [comment]
I'm on master of both, and get the same output as @daschw . MacOS.

**Comment by juliohm** [comment]
Like @pkofod I have the same versions and can't see the image flipped.

**Comment by mkborregaard** [comment]
Could you post `versioninfo()`?
```
Julia Version 0.6.2
Commit d386e40c17 (2017-12-13 18:08 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin14.5.0)
  CPU: Intel(R) Core(TM) i7-5557U CPU @ 3.10GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, broadwell)
```
I don't think this is a Plots/GR version issue.

**Comment by daschw** [comment]
I can't reproduce on 0.17. I get the same result as on master.

**Comment by juliohm** [comment]
```julia
julia> versioninfo()

Julia Version 0.6.2
Commit d386e40c17* (2017-12-13 18:08 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-6500U CPU @ 2.50GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, skylake)

Pkg.status.(["Plots","GR"])

 - Plots                         0.17.0
 - GR                            0.31.0
```

**Comment by mkborregaard** [comment]
I seem to remember you recently rebuilt GR - might you other two try that?

**Comment by juliohm** [comment]
@mkborregaard I did build GR again with `Pkg.build("GR")`, the behavior is still the same. Anything else I could do to help debug the issue?

**Comment by mkborregaard** [comment]
It needs to be `ENV["GRDIR"]=""; Pkg.build("GR")`

**Comment by juliohm** [comment]
Tried with the env variable as above, still same behavior.

**Comment by pkofod** [comment]
Rebuilt, didn't work. It just flips the axis labels, not the values.

**Comment by Paethon** [comment]
Same here

    Julia Version 0.6.3
    Commit d55cadc (2018-05-28 20:20 UTC)
    Platform Info:
      OS: Linux (x86_64-pc-linux-gnu)
      CPU: Intel(R) Core(TM) i5-4200U CPU @ 1.60GHz
      WORD_SIZE: 64
      BLAS: libopenblas (DYNAMIC_ARCH NO_AFFINITY Haswell)
      LAPACK: libopenblas
      LIBM: libm
      LLVM: libLLVM-3.9.1 (ORCJIT, haswell)

    - Plots                         0.17.2
    - GR                            0.31.0

**Comment by Paethon** [comment]
This issue has now been solved in `GR.jl`: https://github.com/jheinen/GR.jl/commit/b16a5da6fd7bca3855156a3e25a5e7556fd2b91c

**Comment by daschw** [comment]
OK, so now yflip stopped working for me on GR:
```julia
julia> versioninfo()
Julia Version 0.6.3
Commit d55cadc350* (2018-05-28 20:20 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i5-7300U CPU @ 2.60GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, broadwell)

julia> foreach(Pkg.status, ("Plots", "GR"))
 - Plots                         0.17.2+            master
 - GR                            0.31.0
```
cc @jheinen

**Comment by Paethon** [comment]
Have you tried the current master of GR? The mentioned fix is not part of 0.31.0
